src/components/Table/Table.tsx

```diff
@@ -480,6 +501,7 @@ const Table = forwardRef<HTMLTableElement, TableProps>(
       onSort,
       loading,
       noDataMessage,
+      size = "sm",
```

Comment:
is this also going to be the value for the unified console?

Reply:
Yes
Most of the places we use is sm and very few places need md so added as default